src/libPMacc/include/particles/ParticlesBase.kernel

```diff
@@ -636,56 +636,119 @@ struct KernelFillGaps
     }
 };
 
+/** delete all particles within an area
+ *
```

Comment:
add extended description:

deletes all particles in an AREA by setting the particle attribute `multiMask_` to zero